cli/default-reporter/src/reporterForClient/reportLifecycleScripts.ts

```diff
@@ -215,15 +216,16 @@ function highlightLastFolder (p: string) {
 
 const ANSI_ESCAPES_LENGTH_OF_PREFIX = hlValue(' ').length - 1
 
-function createStreamLifecycleOutput (cwd: string) {
+function createStreamLifecycleOutput (cwd: string, hideLifecyclePrefix: boolean) {
   currentColor = 0
   const colorByPrefix: ColorByPkg = new Map()
-  return streamLifecycleOutput.bind(null, colorByPrefix, cwd)
+  return streamLifecycleOutput.bind(null, colorByPrefix, cwd, hideLifecyclePrefix)
 }
 
 function streamLifecycleOutput (
   colorByPkg: ColorByPkg,
   cwd: string,
+  hideLifecyclePrefix: boolean,
   logObj: LifecycleLog
 ) {
   const prefix = formatLifecycleScriptPrefix(colorByPkg, cwd, logObj.wd, logObj.stage)
```

Comment:
There is no need to format the prefix if it won't be printed.

Reply:
I left it in there, because the prefix is still needed for the output of the exit code and script name:

```ts
  if (typeof logObj['exitCode'] === 'number') {
    if (logObj['exitCode'] === 0) {
      return `${prefix}: Done`
    } else {
      return `${prefix}: Failed`
    }
  }
  if (logObj['script']) {
    return `${prefix}$ ${logObj['script'] as string}`
  }
```

Only the actual output from the script itself has no prefix.